overrides/scripts/undyeing.zs

```diff
@@ -0,0 +1,102 @@
+//backport 1.14 dye mechanics
+<ore:dyeBlack>.remove(<minecraft:dye:0>);
+<ore:dye>.remove(<minecraft:dye:0>);
+recipes.addShapeless(<metaitem:dye.black>, [<minecraft:dye:0>]);
+
+<ore:dyeBrown>.remove(<minecraft:dye:3>);
+<ore:dye>.remove(<minecraft:dye:3>);
+recipes.addShapeless(<metaitem:dye.brown>, [<minecraft:dye:3>]);
+
+<ore:dyeBlue>.remove(<minecraft:dye:4>);
+<ore:dye>.remove(<minecraft:dye:4>);
+recipes.addShapeless(<metaitem:dye.blue>, [<minecraft:dye:4>]);

```

Comment:
Should add conversions for lazurite, sodalite, and metal mixture dust to their respective dyes as well